### ISSUE TITLE
Error with clipboard.js 1.5.6

### ISSUE THREAD

**Issue by jrjohnson** [jkusa/ember-cli-clipboard#6]
Probably nothing to do with this addon directly, but posting for anyone who looks here first.

There appears to be a problem with the latest version of clipboard.js (1.5.6) https://github.com/zenorocha/clipboard.js/issues/177

Whch looks like
```
TypeError: '[object Object]' is not a constructor (evaluating 'new Clipboard('#' + this.get('elementId'))')
```

Which will be installed automatically as it meets the requirements.  A quick fix is to modify bower.json
from:
`"clipboard": "~1.5.5"`
to:
`"clipboard": "1.5.5"`

**Comment by zenorocha** [comment]
Just released 1.5.7. Everything should be working fine now ;)

**Comment by jkusa** [comment]
@jrjohnson @zenorocha thanks for looking into this!
